lab-michael/app/config/router-config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider) {
+  $urlRouterProvider.when('/', '/home');
+
+  let routes = [
+    {
+      name: 'home',
+      url: '/home',
+      template: require('../view/home/home.html'),
+      controller: 'HomeController',
+      controllerAs: 'homeCtrl'
+    },
+    {
+      name: 'signup',
+      url: '/signup',
+      template: require('../view/signup/signup.html'),
+      controller: 'SignupController',
+      controllerAs: 'signupCtrl'
+    },
+    {
+      name: 'gallery',
```

Comment:
What's this gallery config for? We've not implemented the galleries yet at this point. ;-)